tutorial.md

```diff
@@ -9,49 +9,33 @@ In this tutorial we will go through in detail the steps required to set-up a Pyt
 
-- [the GitHub command line interface (CLI)](https://cli.github.com/),
-- [pre-commit](https://pre-commit.com/),
-- [tox](https://tox.wiki).
+An easy way to install the software you need is by using `uv`, which can create a dedicated environment with the necessary packages installed using the step by step instructions below
 
-These additional three tools are required to complete some of the follow on steps for using the package generated with the template, so while they are not strictly needed, you will get more out of the tutorial if you have them installed. For the follow on exercises you will also need an [account set up on GitHub](https://github.com/join) if you don't already have one.
+1. Open a terminal window
 
-An easy way to get all the software you need installed is using the tool [Conda](https://conda.io/projects/conda/en/latest/user-guide/getting-started.html) to create a dedicated environment with the necessary packages installed by following the step by step instructions below:
-
-1. Download and install [Miniconda](https://docs.anaconda.com/free/miniconda/) following the operating system specific instructions at either
-
-   - <https://docs.anaconda.com/free/miniconda/miniconda-install> for a graphical installer
-   - or <https://docs.anaconda.com/free/miniconda/#quick-command-line-install> for installation via the command line.
-
-   If you already have Miniconda (or Anaconda) installed you can skip this step.
-
-2. Once Miniconda is installed, you need to open a terminal window:
-
-   - On Windows: open the Start menu from the taskbar, type `miniconda` in the search field, then click `Anaconda Prompt (miniconda3)` from the results (or `Anaconda Prompt (anaconda3)` if using a previous Anaconda installation).
+   - On Windows: open the Start menu from the taskbar, type `cmd` in the search field, then click [command prompt](https://en.wikipedia.org/wiki/Command-line_interface#Command_prompt) from the results.
    - On MacOS: click the Launchpad icon in the Dock, type `Terminal` in the search field, then click `Terminal` from the results.
    - On Linux: open the default terminal application installed in your distribution.
 
-3. Once you have a terminal window open you should see text `(base)` as part of your [command prompt](https://en.wikipedia.org/wiki/Command-line_interface#Command_prompt) if Minconda has been installed and set up correctly, indicating you currently have the default `base` environment active. If this is the case you should run
+2. In the terminal window, use
 
    ```sh
-   conda create -y -n python-tooling -c conda-forge cookiecutter git gh pre-commit tox
+   uv tool install cookiecutter
+   uv tool install pre-commit
+   uv tool install tox
```

Comment:
Can these be one line?

Reply:
Not with one `uv` command - `uv tool install --help` gives usage as

```
Usage: uv tool install [OPTIONS] <PACKAGE>
```

and trying with multiple package names gives an error.

It might be possible to stack multiple commands on a line using separators but I'm not sure if this would be easy to do in a way that will work across operating system / shells and also would probably be less readable.

Reply:
We don't need `pre-commit` and `tox` so that should solve it.